jwt/api.py

```diff
@@ -16,7 +16,7 @@
 
 
 class PyJWT(object):
-    def __init__(self, algorithms=None):
+    def __init__(self, algorithms=None, options=None):
```

Comment:
As maintainers it is also easier to commit to a more restricted change to the interface.